NSCC-IT-Web-Capstone-Project-Ola-Nick/capstone-course-grid-development-repo
# Feature/crud lesson

Add the following line at the bottom of the .env file

# wysiwyg api key
TINYMCE_API_KEY=s2itx8rc1ekq634gurfmd8dvpxf3nil7a55iswysiuzgnqw6

npx prisma generate
npx prisma db pull

the sql dump file has been changed, please replace all the content on your branch

Please npm run test

run the program
test UI functionalities:

- add, delete and edit lessons, units
- login for the new credeintials I created user1@gmail.com with password Admin123**. test add crud units and lessons
- Make sure editor in new lesson is working
- make sure the credentials validation works
- copy the address to units, lesson and courses pages from one user, log in with a different user, try to access each of those pages and make sure that you are getting unauthorized pages when accessing those resources.

Note: If something happens, call me
